### ISSUE TITLE
Content deletion results in a 500

### ISSUE THREAD

**Issue by grampelberg** [tensorlakeai/indexify#710]
If I try to delete content that no longer exists, I get a 500. It should be a 404. I'm not sure how we should handle partial failures though, as that API is a bulk API.

**Comment by maxkozlovsky** [comment]
Yes the error handling should be more detailed. It may be not a quick fix though.
